react/src/components/DynamicUnitInputNumberWithSlider.tsx

```diff
@@ -71,9 +86,7 @@ const DynamicUnitInputNumberWithSlider: React.FC<
           max={max}
           units={units}
           // set value to 0mib when min value overs max value
-          value={
-            isMinOversMaxValue(_.parseInt(min), _.parseInt(max)) ? '0m' : value
-          }
+          value={value}
```

Comment:
Doesn't DynamicUnitInputNumber require separation for isMinOversMaxValue?
```suggestion
          value={isMinOversMaxValue ? '0g' : value}
```